SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -99,16 +106,22 @@ struct FConnectionConfig
 		if (LinkProtocolString.Compare(TEXT("Tcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_TCP;
+			OverrideLinkProtocol = true;
+			return;
 		}
 		else if (LinkProtocolString.Compare(TEXT("Kcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_KCP;
+			OverrideLinkProtocol = true;
+			return;
 		}
-		else if (!LinkProtocolString.IsEmpty())
+
+		if (!LinkProtocolString.IsEmpty())
 		{
-			UE_LOG(LogTemp, Warning, TEXT("Unknown network protocol %s specified for connecting to SpatialOS. Defaulting to KCP."),
-				   *LinkProtocolString);
+			UE_LOG(LogTemp, Warning, TEXT("Unknown network protocol '%s' specified for connecting to SpatialOS."), *LinkProtocolString);
 		}
+
+		UE_LOG(LogTemp, Display, TEXT("No link protocol set. Defaulting to TCP for server workers, KCP for client workers."));
```

Comment:
Hmmm, just realised. Why is this using LogTemp and not a proper log class. Also the tests are now spamming `LogTemp: Display: No link protocol set. Defaulting to TCP for server workers, KCP for client workers.`. I think it's unnecessary to have this set to Display. Let's make it verbose.

Reply:
Is setting a custom log class possible when there is no `.cpp` (only the header)? Can't find much documentation on this